### DISCUSSION TITLE
Is there anything new for Minimal API in the upcoming net8?

### DISCUSSION THREAD

**Discussion by ali50m** [dotnet/aspnetcore#51346] [Q&A] [answered]
I have not read such related topics over the internet.

**Comment by martincostello** [comment, marked as answer]
Here are at least two mentions of Minimal API changes for .NET 8:

- [Support for form files in new form binding](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-rc-2/#support-for-form-files-in-new-form-binding)
- [Support for keyed services in minimal APIs, MVC, and SignalR](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-rc-1/#apis)

Check out the various blog posts from the ASP.NET Core team over the release cycle [since Preview 1 back in February](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-preview-1/#apis) for more information about what's new for Minimal APIs in .NET 8, as well as upcoming talks [at .NET Conf](https://www.dotnetconf.net/agenda).

**Reply by ali50m** [reply]
thanks!
